tests/featureTesting/compare_json.py

```diff
@@ -0,0 +1,149 @@
+import constants as cs
+
+import helper_functions as hf
+
+import pandas as pd
+
+
+class CompareJsons(object):
```

Comment:
Add documentation for class as well as the function.

Reply:
class docs?

Reply:
I have updated the code. I think you are not able to see the changes done after sending the PR.